e2e/cluster/local/images/k3s/Dockerfile

```diff
@@ -1,12 +1,11 @@
 # Install the all-in-one binary so we can copy our run-time images into the image
 # which helps avoid pulling them when running e2e tests.
-ARG ALPINE="library/alpine:3.14"
-FROM ${ALPINE} AS k3s
+ARG SLES="registry.suse.com/suse/sle15:15.3"
```

Comment:
Why SLES here instead of BCI? Unpacked on my system, the SLES image is 7 times larger than Alpine.

Reply:
In fact it looks like `bci-base` and `sle15` images are identical.
If you check here, they have the same digest:
https://registry.suse.com/static/suse/sle15sp3/index.html
https://registry.suse.com/static/bci/bci-base/index.html

But I agree it is unfortunate that they are so large.

There are also `bci-minimal` and `bci-micro` images which are small but they are still in Tech Preview.
As I understand, @macedogm is preparing a proposal to use them with sidecars but I don't know if we can use that yet.

Reply:
I'm discussing internally about which image to "standardize". I don't believe that we will follow with the idea of the sidecar, but actually with a mix of `bci-micro` and `bci-base/sle15` (when really needed). Let's see how it'll progress.

Reply:
`registry.suse.com/suse/sle15:15.3` is sle-bci, not full SLES.